### PR TITLE
Release 0.2.2

### PR DESCRIPTION
**Pull request by AntonReinhard**
Release 0.2.2

### CODE REVIEW

**Comment by AntonReinhard** [comment]
Do **NOT** squash this PR

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+[diff since 0.2.0](https://github.com/QEDjl-project/QEDbase.jl/compare/release-0.2.0...release-0.2.2)
+
+This release adds some convenience overloads to existing functions, some code maintenance and small fixes.
+
+### Breaking Changes
+
+This release removes the compute setup interface completely since it was deprecated already.
+See https://github.com/QEDjl-project/QEDbase.jl/pull/91 for details.
+
+### New features
```
Comment (szabo137) [inline CHANGELOG.md:14]: Would you mind adding references to the respective PR for the changes? The milestone `Release-v0.2.2` might come in handy.

**Review by szabo137 — APPROVED** [review]
One minor comment on the changelog. The rest is looking good for me.

File: CHANGELOG.md (hunk 1 above)
Comment (szabo137) [reply CHANGELOG.md:14]: nice! Thanks.

**Review by szabo137 — APPROVED** [review]
No additional concerns from my side. Feel free to proceed, if the pipeline succeeds. 

And don't forget: in the case of merging

160581061-d6e29c7f-8ed4-44d0-8d64-872d836a9647

**Pull request merged**